app/src/main/res/layout/activity_main.xml

```diff
@@ -42,7 +42,9 @@
             app:stepperTextColor="@color/stepper_text_blue"
             app:stepperActionsColor="@color/stepper_text_blue"
             app:stepperActionsDisabledColor="@color/stepper_actions_blue"
-            app:stepperBackgroundColor="@color/stepper_background_blue"/>
+            app:stepperBackgroundColor="@color/stepper_background_blue"
+            app:stepperAllowNegativeStepper="false"
```

Comment:
Almost there, only thing that needs to be fixed is this! `stepperAllowNegative` instead of `stepperAllowNegativeStepper`. Currently it makes the build fail